packages/verified-fetch/README.md

```diff
@@ -522,7 +522,7 @@ This module attempts to act as similarly to the `fetch()` API as possible.
 
 This library supports the following methods of fetching web3 content from IPFS:
 
-1. IPFS protocol: `ipfs://<cidv0>` & `ipfs://<cidv0>`
+1. IPFS protocol: `ipfs://<cidv0>` & `ipfs://<cidv1>`
```

Comment:
Please also update https://github.com/ipfs/helia-verified-fetch/blob/3d8bb789aa68374ca79ea9bac0931350390b141e/packages/verified-fetch/src/index.ts#L496 where this is generated